spec_3/index.js

```diff
@@ -0,0 +1,34 @@
+let example = "Catch me outside, how about that?"
+
+const dropEmpty = (array) => {
+  console.log(array)
+  let newArray = []
+  for(i = 0; i < array.length; i++){
```

Comment:
When I see the pattern: variable collection declaration, for loop, return declared variable, this stands out to me as an opportunity for more concise code using functional methods on the collection object:
```js
const dropEmpty = array =>
  array.filter( word => word !== undefined )
```